tests_v2/unit/app/routes/political/id_lookup/test_id_lookup.py

```diff
@@ -0,0 +1,306 @@
+from typing import Any, Dict, List, Optional
+
+import pytest
+from fastapi import HTTPException
+from httpx import AsyncClient
+
+from app.models.pydantic.geostore import GeostoreCommon
+from app.routes.political import id_lookup
+from app.routes.political.id_lookup import _admin_boundary_lookup_sql, normalize_names
+
+ENDPOINT_UNDER_TEST = "/political/id-lookup"
+
+
+@pytest.mark.asyncio
+async def test_sanitize_names_pass_through() -> None:
```

Comment:
I don't want to hold up this merge. 
So, I'm fine with a follow-up PR that refactors the test descriptions to remove `sanitize`.

Reply:
D'oh! Thanks for pointing this out. Yup, will fix soon.